pnpm-workspace.yaml

```diff
@@ -1,5 +1,6 @@
 packages:
   - 'projects/*/*'
+  - '!projects/js-packages/jetpack-cli'
```

Comment:
🤔 Interesting...

This means it won't be runnable directly from a monorepo checkout. But maybe you could somehow point `npm -g` at the monorepo checkout as a source when testing?

And hopefully eslint doesn't get confused by missing imports, although I suppose we could probably just disable the relevant rules.

Reply:
Nod, with having it in the workspace, `pnpm cli-setup` would pick up on it and globally link `jp` to the copy in that monorepo. Then, when you nerf `node_modules` (or delete that checkout totally) and want to use `jp init` to start again, no joy.

It might be something we can adjust in another way, but if we keep `jp` to be relatively stable (most everything it does fires code within the monorepo's pnpm install), it might not be horrible to have it be annoying to test changes within the monorepo? Or, you can `npm install` within the `projects/js-packages/jetpack-cli`dir and run `node projects/js-packages/jetpack-cli/bin/jp.js [cmd]` (how I did most of my testing).

Reply:
I wasn't too worried about the potential difficulty in testing. But documenting how to test it in the js-package's README.md might be a good idea.